recipes/qarchive/all/conanfile.py

```diff
@@ -0,0 +1,110 @@
+import os
+from conans import ConanFile, tools, CMake
+from textwrap import dedent
+
+class qarchiveConan(ConanFile):
+    name = "qarchive"
+    license = "BSD-3-Clause"
+    homepage = "https://antonyjr.in/QArchive/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = ("QArchive is a cross-platform C++ library that modernizes libarchive ,\
+                    This library helps you to extract and compress archives supported by libarchive")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    topics = ("conan", "qt", "Qt", "compress", "libarchive")
+    exports_sources = ["CMakeLists.txt"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.configure()
+        return self._cmake
+
+    def _patch_sources(self):
```

Comment:
This is ugly but easier to maintain than a patch file. I don't mind porting it to a patch file (once the changes are approved) if the community prefers it that approach.